src/authoring-docs.md

```
@@ -49,9 +47,13 @@ The following file
-<limel-example-button-group/>
+<limel-example-button-group-basic/>
 ```
 
 will generate the following
 
-<limel-example-button-group />
+<limel-example-button-group-basic />
+
+## Sending the changes our way
+
+Once your documentation change is ready, head over to [Commits & Pull Requests](#/Home/commits-and-prs.md/) to learn how we like commits to be shaped and how to open a PR.
```

markdownlint-cli2 (0.22.1)

[warning] Link fragments should be valid

(MD051, link-fragments)